modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -40,6 +40,16 @@ A user requires the following privileges to install an {product-title} cluster:
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-vmotion_{context}"]
+== Using {product-title} in vMotion enabled vSphere clusters
```

Comment:
@gnufied Doesn't this seem like a very round about way to say vmotion is supported but storage vmotion is not?

With the continuing issues I think we should be explicit. 

"Storage vMotion is not supported. Since Storage DRS uses Storage vMotion it is also not supported.
If Storage vMotion is used there is a potential for data loss."

Reply:
Sure. But we still should mention that compute only vmotion works though.

Reply:
@gnufied I completely agree that it should be noted that vmotion (compute) is supported since that has been a question floating around.

Reply:
I reworded this to - `hence any form of *storage vMotion* is unsupported.`. But I also wanted to ephasis that, people shouldn't be moving their VMs/VMDKs manually or use datastores that are part of datastore cluster.

Reply:
I do think an explicit warning about the potential for data loss could be beneficial here to reinforce the risk.

Reply:
I made a few wording changes to apply docs style guidelines before encountering these comments. I will make another update in a moment to incorporate these suggestions.

Reply:
I added wording about data loss and wiped away @bobfuru changes. I am done now though. please feel free to modify it.

Reply:
I worked with @gnufied to make minor tweaks for docs conventions.

Reply:
This will also need to be CP'ed to `enterprise-4.8` branch when that is opened.

Reply:
Cc: @kmccarron-rh This fixes 4.x but will need to also be somehow introduced in OCP 3.11 docs as part of https://bugzilla.redhat.com/show_bug.cgi?id=1850245.